src/content/docs/apm/agents/php-agent/getting-started/introduction-new-relic-php.mdx

```diff
@@ -146,7 +146,7 @@ After installing the agent, go further and extend the agent's instrumentation:
 
 ## Troubleshoot your installation [#troubleshoot]
 
-If you encounter issues with the PHP agent, see our [full list of troubleshooting documentation](/docs/agents/php-agent/troubleshooting). Common installation issues include:
+If you're having problems, see the PHP agent troubleshooting docs. Common issues include:
```

Comment:
I prefer what you've got in the Node.js doc: "Some of the most important troubleshooting docs in that section include:"